documentation/maintain_and_tune.markdown

```diff
@@ -15,15 +15,13 @@ canonical: "/puppetdb/latest/maintain_and_tune.html"
 [logback]: ./configure.html#the-logback-logging-config-file
 [dashboard]: #monitor-the-performance-dashboard
 
-PuppetDB requires a relatively small amount of maintenance and tuning. You should become familiar with the following occasional tasks:
+PuppetDB requires a relatively small amount of maintenance and tuning. However, you should become familiar with the occasional tasks outlined in this guide.
 
-## Monitor the Performance Dashboard
+## Monitor the performance dashboard
 
-Once you have PuppetDB running, visit this URL, substituting the name
-and port of your PuppetDB server:
-`http://localhost:8080/pdb/dashboard/index.html`
+Once PuppetDB is running, visit `http://localhost:8080/pdb/dashboard/index.html`, substituting the name and port of your PuppetDB server. 
```

Comment:
the full path isn't needed -- 
visit http://localhost:8080 in your browser, substituting...